### PR TITLE
Add network field to wallet record on localStorage

### PR DESCRIPTION
**Pull request by iambeone**
Network field needed to display account network and to switch accounts in different networks.
This pr is not needed if we will decide to identify the network by address prefix.

### CODE REVIEW

File: test/keystore.spec.ts [hunk 1]
```diff
@@ -25,6 +28,15 @@ describe(`Keystore`, () => {
     ).toBeDefined()
   })
 
+  it(`check network is set`, () => {
+    storeWallet(mockWallet4, 'mock-name4', 'mock-password', 'regen-testnet')
+    expect(
+      localStorage.getItem(`cosmos-wallets-xrn:1h0y77r8ee28hs0wqg9css7rzegmagaamwl6rdp`)
+    ).toBeDefined()
+    const wallet = JSON.parse(localStorage.getItem(`cosmos-wallets-xrn:1h0y77r8ee28hs0wqg9css7rzegmagaamwl6rdp`) || '[]')
```
Comment (faboweb) [inline test/keystore.spec.ts]: why are you defaulting to an array here?

```suggestion
    const wallet = JSON.parse(localStorage.getItem(`cosmos-wallets-xrn:1h0y77r8ee28hs0wqg9css7rzegmagaamwl6rdp`) || '{}')
```

File: src/cosmos-keystore.ts [hunk 2]
```diff
@@ -86,13 +86,14 @@ function loadFromStorage(address: string): StoredWallet | null {
 }
 
 // stores an encrypted wallet in localstorage
-function addToStorage(name: string, address: string, ciphertext: string): void {
+function addToStorage(name: string, address: string, ciphertext: string, network?: string): void {
```
Comment (faboweb) [inline src/cosmos-keystore.ts]: should this be optional? I would propose we make this mandatory to keep the API clean.

Comment (iambeone) [reply src/cosmos-keystore.ts]: I think as we have this module in npm we should add backward compatibility to our code.

Comment (faboweb) [reply src/cosmos-keystore.ts]: Aren't for this case semver versions? The API changes so we bump the minor version?

**Comment by faboweb** [comment]
Changelog is missing

File: src/cosmos-keystore.ts (hunk 2 above)
Comment (iambeone) [reply src/cosmos-keystore.ts]: Updated)

**Pull request merged**